pages/enterprise/themes.rst

```diff
@@ -0,0 +1,47 @@
+###############################
+Custom Themes and Notifications
+###############################
+
+************
+Introduction
+************
+
+The Theme Customizer for Graylog Enterprise gived you visual control of your instance. Consider ways it's used
+to meet your needs. Administrators can:
```

Comment:
Not really sure what the "Consider ways it's used to meet your needs." sentence is really doing there.  Also, typo on "gived" should be "gives".

```suggestion
The Theme Customizer for Graylog Enterprise gives you control over the color scheme of your instance. Administrators can:
```

Reply:
Yeah, that definitely leans toward blog-writing. I'll just remove the sentence, and fix that typo.

Reply:
I like it. Updated.